packages/cli/__tests__/zosfiles/__unit__/create/like/ds.handler.unit.test.ts

```diff
@@ -0,0 +1,100 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { Create, CreateDataSetTypeEnum } from "@zowe/zos-files-for-zowe-sdk";
+import { UNIT_TEST_ZOSMF_PROF_OPTS } from "../../../../../../../__tests__/__src__/mocks/ZosmfProfileMock";
+
+describe("Create data set handler", () => {
+    describe("process method", () => {
+        it("should create a  data set if requested", async () => {
+            // Require the handler and create a new instance
+            const handlerReq = require("../../../../../src/zosfiles/create/classicPds/ds.handler");
```

Comment:
Please update the directory in path for your new handler: "../../../../../src/zosfiles/create/ds/ds.handler" (after renaming "create/like" to "create/ds" as mentioned below).